application/models/services/SurveyDeactivate.php

```diff
@@ -236,7 +236,7 @@ protected function handleSurveyTable($iSurveyID, $date, &$aData, $userID, $DBDat
         }
 
         $this->app->db->createCommand()->renameTable($sOldSurveyTableName, $sNewSurveyTableName);
-        $this->archiveTable($iSurveyID, $userID, "old_tokens_{$iSurveyID}_{$date}", 'response', $DBDate, json_encode(Response::getEncryptedAttributes($iSurveyID)));
+        $this->archiveTable($iSurveyID, $userID, "old_survey_{$iSurveyID}_{$date}", 'response', $DBDate, json_encode(Response::getEncryptedAttributes($iSurveyID)));
```

Comment:
Need to be an array here ? No ?

Reply:
This is the `properties` parameter. That wasn't updated.

Reply:
Oups, right … string